app/helpers/webpack_helper.rb

```diff
@@ -0,0 +1,16 @@
+module WebpackHelper
+  def webpack_assets_path(path)
+    return "http://0.0.0.0:3333/#{path}" if Rails.env.development?
+
+    host = Rails.application.config.action_controller.asset_host
+    manifest = Rails.application.config.assets.webpack_manifest
+    path = manifest[path] if manifest && manifest[path].present?
+    "#{host}/assets/#{path}"
+  end
+
+  def webpack_css_tag(filename)
+    if !Rails.env.development?
```

Comment:
__[RuboCop]__

Style/NegatedIf: Favor `unless` over `if` for negative conditions.


<!-- Comment by [SideCI](https://sideci.com) -->